lib/modules/manager/kustomize/extract.spec.ts

```diff
@@ -452,6 +469,20 @@ describe('modules/manager/kustomize/extract', () => {
         ],
       });
     });
+
+    it('parses helmChart field with oci registry', () => {
+      const res = extractPackageFile(kustomizeOciHelmChart);
+      expect(res).toMatchSnapshot({
+        deps: [
+          {
+            depType: 'Docker',
```

Comment:
This was the only part I was uncertain. Is this the correct value for docker?

Reply:
depType here isn't really necessary because users can look/filter based on datasource. But seeing as depType=HelmChart is already in use then I think that the way you chose is probably best for consistency